docs/User-Guide_Networking.md

```diff
@@ -1,17 +1,19 @@
-# Networking
+## Networking
 
 Armbian uses [**Netplan.io**](https://netplan.io/) to describe networking configurations. Netplan is a utility to easily configure Linux networking, using a declarative approach.
 If you want to configure your network manually, it is as simple as editing and creating Netplan yaml files (see the yaml configuration reference at the [Netplan docs](https://netplan.readthedocs.io/en/stable/netplan-yaml/)).
 
 Netplan is used to configure networks on **all** Armbian images since Release 24.05, no matter if minimal or desktop, Debian or Ubuntu. However, the networking backends are different based on if you choose a minimal image or not. 
```

Comment:
It his correct? AFAIR minimal was systemd-networkd only?

Reply:
Minimal is by default systemd-networkd, yes. Perhaps changing to:

"Netplan is used to configure networks on **all** Armbian images, no matter if minimal or desktop, Debian or Ubuntu. However, the networking backend on minimal is `systemd-networkd` which has a very small footprint, runs much faster and thus contribute  to leaner user experience"